src/composable/useErrorHandling.ts

```diff
@@ -93,16 +93,18 @@ export const useErrorQueue = (): {
     errors.value = [];
   };
 
+  // TODO if used across different components, only valibe on first run, then reactivty is broken
```

Comment:
typo

Reply:
Erledigt in https://github.com/MORE-Platform/more-studymanager-frontend/issues/529